windows-installer/renew-up.ps1

```diff
@@ -0,0 +1,12 @@
+$taskName = "Certbot Renew Task"
```

Comment:
I saw the Certbot renew task get added and removed from the task scheduler, but trying to run the task manually to test it didn't work. This may be due to the typo in the NSIS template below.

Reply:
This is working fine now.

Reply:
It still doesn't seem to be working for me. Have you tested it with the latest changes?

When I go into task scheduler, enable history, and run the task, the history contains:
```
Task Scheduler successfully completed task "\Certbot Renew Task" , instance "{07e12b3b-4022-4737-8543-4b32704db640}" , action "Powershell.exe" with return code 2147942401.
```
That return code is illegal function according to https://stackoverflow.com/questions/48343993/batch-file-from-scheduled-task-returns-code-2147942401/51499918.

Additionally, there should be a Certbot log file created from the run which wasn't happening for me.

Reply:
Indeed, it was working for me because I run the installation in "AllUsers" mode while you are using the "CurrentUser" mode. In your case, the scheduled task fails, because it is run under the "SYSTEM" account while certbot is accessible only to your account.

I updated the creation of the scheduled task to take into account the current installation mode:
* if it is "AllUsers", create the task to be run under "SYSTEM" account
* if it is "CurrentUser", create the task to be run under current user account

I also improved two things:
* use the MultiUser NSIS extension to require the user level (Admin), since it will handle it more  finely than the NSIS buit-in option.
* propose the "AllUsers" installation mode by default, since it should be the widest case for a system tool like certbot.

Reply:
This is in response to https://github.com/certbot/certbot/pull/7371#discussion_r325060131.

That makes sense.

Out of curiosity, what do you think about the idea of removing this choice for users for now and forcing Certbot to be installed for all users? Is this easy to do? Is it a bad idea?

My motivation for just wondering about this that a simpler installer that supports fewer setups initially makes maintenance and debugging easier and we could add features like this if they're needed. If this isn't easy to do or  there's a good reason users actually need this functionality, I think it's not worth worrying about.

Reply:
Well, I share your concern.

So, is it simple? Yes. Since now the default is to make a global installation when the installer is run as admin, and since the installer can be run only as admin, it is just a matter of removing the installation mode interface. Three lines to edit. 

Is it bad? At least it is not a modern approach. Since Windows Vista you can have tools installed in user space, but it is only since very few years that I see tools that uses this capability in their design. It is really interesting in a server context, because you can have a contained approach: it is possible to have several instances of a program, in different versions and states, available specifically on different users.

This is a desirable thing for an admin sys, and that is why I was initially reluctant to remove it for Certbot. 

However this approach works really if the program does not require to be run as admin, and if it is designed to have an isolated state for the user (eg. a config folder in the user home). First is not possible, and second is difficult with Certbot currently.

So until Certbot is really designed for that, and that there is a real need for user space executions of Certbot, it is certainly better for now to just allow a global installation.